corehq/apps/es/es_query.py

```diff
@@ -244,13 +246,35 @@ def _assemble(self):
                 facet.name: {facet.type: facet.params}
                 for facet in self._facets
             }
+        if self._source:
+            self.es_query['_source'] = self._source
 
     def fields(self, fields):
-        """Restrict the fields returned from elasticsearch"""
+        """
+            Restrict the fields returned from elasticsearch
+
+            Usage Note: As of ES 1.x, fields will only work on leaf nodes! It will no longer return an object,
+            e.g. field.*, to return an object refer to '.source'
+            """
```

Comment:
Should we just switch everything to use `_source`?  The [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-fields.html) make it sound like that's the preferred approach (see warning at the top).  It also feels more in-line with our usage.  @snopoke thoughts?